### DISCUSSION TITLE
Thorium Nova Base Features

### DISCUSSION THREAD

**Discussion by alexanderson1993** [Thorium-Sim/thorium-nova#157] [Ideas]
This is going to be a place to propose possible features for Thorium Nova. I want this to be a brainstorm discussion, where any and all ideas are welcome. Each item will eventually be turned into its own discussions where we can propose and discuss how those features will behave and be implemented. This is where we will whittle down the list of features.

What counts as a feature? Here are some possibilities, categorized, but with a lot of overlap.
- Core functionality, like missions, flights, the ship map, the star map, stations, the view screen, and the flight director screen. We've already started some discussions about those that I'll link here (eventually)
- Plugin aspects, or ways that Thorium Nova can be customized. This includes star maps, ship systems, missions,
- Ship systems, like engines, weapons, communications, etc.
- Crew cards used by the crew to interact with the ship, its systems, its crew, and the objects around it.
- Flight director core controls
- Miscellaneous features that can be used by the crew or flight director to make the flight more enjoyable, easier to run, and more engaging.

So, what kinds of features are you interested in having implemented first?

**Comment by astrolamb-gaming** [comment]
Personally I think that the core flight director controls should probaby be pretty high on our priority list here. As we add other features, we should add applicable functionality to the flight director controls as well. 
Immediately following the very basic flight director stuff, I think I would recommend implementing missions/flights and some basic crew cards so that we can start testing integrated features.
